locale/ForemanDocker.pot

```diff
@@ -0,0 +1,633 @@
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/registries_controller.rb:48
+msgid "Delete a docker registry"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:13
+msgid "List all containers"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:15
+msgid "List all containers in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:29
+msgid "Show a container"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:31
+msgid "Show container in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:43
+msgid "Registry this container will have to\\n                                                        use to get the image"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:46
+msgid "Name of the repository to use\\n                                                       to create the container. e.g: centos"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:49
+msgid "Tag to use to create the container. e.g: latest"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:60
+msgid "The capsule this container will have to use\\n                                                       to get the image. Relevant for images\\n                                                       retrieved from katello registry."
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:66
+msgid "Create a container"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:68
+msgid "Create container in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:84
+msgid "Wrong attributes: %s"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:88
+msgid "Delete a container"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:90
+msgid "Delete container in a compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:98
+msgid "Show container logs"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:100
+msgid "Show logs from a container in a compute resource"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:105
+msgid "Number of lines to tail. Default: 100"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:114
+msgid "Run power operation on a container"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:116
+msgid "Run power operation on a container in a compute resource"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/controllers/api/v2/containers_controller.rb:121
+msgid "power action, valid actions are (start), (stop), (status)"
```

Comment:
Shouldn't use brackets to quote valid values, in fact since this is in API docs, it should pass an array of valid values in the `param` instead of `String`.